exporter.go

```diff
@@ -819,6 +863,18 @@ func (e *Exporter) extractTile38Metrics(ch chan<- prometheus.Metric, c redis.Con
 	}
 }
 
+func (e *Exporter) extractConnectedClientMetrics(ch chan<- prometheus.Metric, c redis.Conn) {
+	if reply, err := redis.String(doRedisCmd(c, "CLIENT", "LIST")); err == nil {
+		clients := strings.Split(reply, "\n")
+
+		for _, c := range clients {
+			if id, host, port, name, age, idle, flags, db, cmd, ok := parseClientListString(c); ok {
+				e.registerConstMetricGauge(ch, "connected_clients_details", id, host, port, name, age, idle, flags, db, cmd)
```

Comment:
Don't set the value of the gauge to `id`, set it to `1.0` (and you can either discard `id` or return it as a string from `parseClientListString() and use it (but if you use it you need to add it as a label in 309) - no strong preference).

In that way you can use `sum()` in PromQL to figure out how many clients per IP, name, etc you have.

Reply:
OHHHHHHH why didn't you say something earlier 🤔

Reply:
sooooooooooooooooooorry !

Reply:
@oliver006 That's what i was struggling with the whole time

Reply:
:-) nice work getting this to work!